Tests/ClasslibraryAsSdk/ClasslibraryAsSdk.csproj

```diff
@@ -5,11 +5,18 @@
 	<PropertyGroup>
 		<!--<TargetFramework>uap10.0</TargetFramework>-->
 		<!--<TargetFrameworks>uap10.0;uap10.0.16278;monoandroid70;xamarin.ios10;xamarin.mac20</TargetFrameworks>-->
-    <TargetFrameworks>uap10.0;uap10.0.16299;monoandroid70;xamarin.ios10;xamarin.mac20;tizen40</TargetFrameworks>
-    <IsPackable>false</IsPackable>
+    <TargetFrameworks>netstandard2.0;net46;uap10.0;uap10.0.16299;monoandroid70;xamarin.ios10;xamarin.mac20;tizen40</TargetFrameworks>
+    <RuntimeIdentifiers Condition="'$(TargetFramework)' == 'netstandard2.0'">win;unix</RuntimeIdentifiers>
```

Comment:
@onovotny I'm looking to figure update the configuration system used in http://github.com/dotnet/corefx and I've been debating whether or not to overload TargetFrameworks/RuntimeIdentifiers or just keep doing what we already do and create a list of BuildConfiguration/PackageConfiguration (see https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/Configurations.props as example) and continue doing our own custom multiplexing. 

Is there a particular reason you decided to continue to use TargetFrameworks/RuntimeIdentifiers?

Reply:
Mainly because it seemed like the natural fit that matched what the goal had to be in the NuGet package? Happy to discuss other/better ways. The key goal was to have the Build action build all of them at once so that Pack could get the outputs and package them.